v2/commands/job/logs.go

```diff
@@ -0,0 +1,92 @@
+package job
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	oapi "github.com/rescale-labs/htc-cli/v2/api/_oas"
+	"github.com/rescale-labs/htc-cli/v2/common"
+	"github.com/rescale-labs/htc-cli/v2/config"
+	"github.com/rescale-labs/htc-cli/v2/tabler"
+)
+
+func logs(ctx context.Context, c oapi.JobInvoker, projectId, taskId, jobId, pageIndex string) (*oapi.HTCJobLogs, error) {
+	res, err := c.GetLogs(ctx, oapi.GetLogsParams{
+		ProjectId: projectId,
+		TaskId:    taskId,
+		JobId:     jobId,
+		PageSize:  oapi.NewOptInt32(pageSize),
+		PageIndex: oapi.NewOptString(pageIndex),
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	switch res := res.(type) {
+	case *oapi.HTCJobLogs:
+		return res, nil
+	case *oapi.GetLogsUnauthorized,
+		*oapi.GetLogsForbidden:
+		return nil, fmt.Errorf("forbidden: %s", res)
+	}
+	return nil, fmt.Errorf("Unknown response type: %s", res)
```

Comment:
This is not really the best - sorry it's elsewhere in the code to pattern off.

I think you want to print the actual type:

```suggestion
	return nil, fmt.Errorf("Unknown response type: %T", res)
```